ember-app/app/templates/settings/integrations/hipchat.hbs

```diff
@@ -9,6 +9,10 @@
     <label>
       {{input value=model.authToken placeholder="Authorization Token" required=true }}
     </label>
+    <p> HipChat Endpoint (no trailing slashes): </p>
+    <label>
+      {{input value=model.endpoint required=true }}
```

Comment:
The value is pre filled as the default HipChat API endpoint `"https://api.hipchat.com"` as per `ember-app/app/controllers/settings/integrations.js` Line 8, are you suggesting even more examples ?